### PR TITLE
Add 3d layout methods

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: pyaedt/modeler/Model3DLayout.py [hunk 1]
```diff
@@ -105,6 +105,63 @@ def obounding_box(self):
         """ """
         return self.oeditor.GetModelBoundingBox()
 
+
+    @aedt_exception_handler
+    def colinear_heal(self, selection, tolerance=0.1):
+        """
```
Comment (akaszynski) [inline pyaedt/modeler/Model3DLayout.py:111]: Would be helpful to have a short summary on both these methods.  Docs has will have no idea what these do.

Comment (maxcapodi78) [reply pyaedt/modeler/Model3DLayout.py]: Improved Docstrings. Added short description and example

File: pyaedt/modeler/Model3DLayout.py [hunk 2]
```diff
@@ -105,6 +105,86 @@ def obounding_box(self):
...
+        self.oeditor.Heal(["NAME:Repair", "Selection:=", selection, "Type:=", "Colinear", "Tol:=",
+                           self.primitives.arg_with_dim(tolerance)])
+        return True
+
+
+
+    @aedt_exception_handler
+    def expand(self, object_to_expand,  size=1, expand_type="ROUND", replace_original=False):
+        """Expand the object by a specific size. If replace_original is ``False`` the method will create a new object
+
```
Comment (akaszynski) [inline pyaedt/modeler/Model3DLayout.py:146]:
```suggestion
        """Expand the object by a specific size.

        If replace_original is ``False`` the method will create a new object.

```
Short description should always be one sentence.  Longer description should be added below.

**Review by akaszynski — APPROVED** [review]
One other suggestion, but otherwise looks good to me.

**Comment by PipKat** [comment]
@maxcapodi78 
Oops--Thought this was my PR and so accepted Alex's docstring change. for you. Sorry!

**Comment by maxcapodi78** [comment]
no problem. I would have done

**Pull request merged**